solc/CommandLineInterface.cpp

```diff
@@ -995,6 +996,8 @@ void CommandLineInterface::handleCombinedJSON()
 				contractData[g_strFunDebugRuntime] = StandardCompiler::formatFunctionDebugData(
 					m_assemblyStack->runtimeObject(contractName).functionDebugData
 				);
+			if (m_options.compiler.combinedJsonRequests->ethdebug)
+				contractData[g_strEthdebug] = Json::object();
```

Comment:
Combined JSON has been deprecated for ages now (#10278) and we're likely to drop it in the next breaking release. It probably does not make sense add more things to it at this point.